### PR TITLE
chore(npm): Refactor npm scripts, move babel into dev deps

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Also add usage instructions for the browser in README.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,18 +2,21 @@
...
+  ],
   "scripts": {
-    "babel": "babel src -d lib",
-    "build": "npm run clean && npm run babel",
+    "build-lib": "babel src -d lib",
+    "build": "npm run clean && npm run build-lib",
     "clean": "rm -rf dist/* && rm -rf lib/*",
-    "webpack": "webpack --progress --colors --optimize-minimize --optimize-occurrence-order --optimize-dedupe",
-    "dist": "npm run build && npm run webpack",
-    "postinstall": "npm run build",
```
Comment (dmitrizagidulin) [inline package.json:15]: The important bit here is the removal of this `postinstall` trigger. And the addition of the `files:` section.

File: webpack.config.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ module.exports = {
     libraryTarget: 'var'
   },
   externals: {
+    'text-encoding': 'TextEncoder',
```
Comment (dmitrizagidulin) [inline webpack.config.js:14]: Almost forgot this bit. If I remember correctly, `text-encoder` was the node shim for the in-browser `TextEncoder` object. Added this line to reflect that.

Comment (dmitrizagidulin) [reply webpack.config.js:14]: And, amusingly enough, the size of the `jose.min.js` webpack'd bundle went from 605K (with the `text-encoder` lib bundled in) to a svelte 69K. :)

Comment (christiansmith) [reply webpack.config.js:14]: Actually, text-encoder is also a shim for browsers that don't support the api natively.

Comment (christiansmith) [reply webpack.config.js:14]: It should work in (I think) Chrome and FF without this dependency. We were looking at some other packages that are slimmer to replace it. Needs to be fully evaluated.

**Pull request merged**